_maps/shuttles/shiptest/bar_ship.dmm

```diff
@@ -2282,9 +2282,7 @@
 /obj/machinery/door/firedoor/border_only{
 	dir = 8
 	},
-/obj/machinery/door/airlock/freezer{
-	name = "freezer airlock"
-	},
+/obj/machinery/door/airlock/freezer,
```

Comment:
Any reason it touches this file?

Reply:
no, actually. i didn't touch it at all, it might have been an odd pull? not really sure